projectgenerator/libqgsprojectgen/dbconnector/db_connector.py

```diff
@@ -58,6 +58,7 @@ def get_tables_info(self):
                 type  (Geometry Type)
                 kind_settings
                 ili_name
+                extent (insert coordinates xmin, ymin, xmax, ymax)
```

Comment:
needs to be semicolon, not comma separated, reading the code?

Reply:
Right, done.